<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <property name="windowOpacity">
   <double>0.958000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,stop: 0 rgb(72, 85, 99), stop: 1 rgb(41, 50, 60));
color:white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(0,0,0,0%);
color:red;</string>
       </property>
       <property name="text">
        <string>Task Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameField">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(219, 219, 219);
color:black;</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Maximum length 50 characters</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(0,0,0,0%);
color:red;</string>
       </property>
       <property name="text">
        <string>Task Descriptions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="DescriptionField">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(219, 219, 219);
color:black;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="styleSheet">
        <string notr="true">background-color:red;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../Bangla-Voice-Calculator/appResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
